qiskit/visualization/bloch.py

```diff
@@ -290,7 +290,7 @@ def set_label_convention(self, convention):
             self.zlabel = ["$\\circlearrowleft$", "$\\circlearrowright$"]
             self.xlabel = ["$\\leftrightarrow$", "$\\updownarrow$"]
         else:
-            raise Exception("No such convention.")
+            raise NotImplementedError("No such convention.")
```

Comment:
This is maybe a `ValueError` because it refers to being given an incorrect string.